examples/data-solutions/dp-foundation/01-lnd-datastorage.tf

```diff
@@ -0,0 +1,93 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  lnd_bucket_retention_policy = {
+    retention_period = 7776000 # 90 * 24 * 60 * 60
+    is_locked        = false
+  }
+}
+###############################################################################
+#                                     GCS                                     #
+###############################################################################
+
+module "lnd-sa-cs-0" {
+  source     = "../../../modules/iam-service-account"
+  project_id = module.lnd-prj.project_id
+  name       = "cs-0"
+  prefix     = local.prefix_lnd
+  iam = {
+    "roles/iam.serviceAccountTokenCreator" = [
+      local.groups_iam.data-engineers
+    ]
+  }
+}
+
+module "lnd-cs-0" {
+  source        = "../../../modules/gcs"
+  project_id    = module.lnd-prj.project_id
+  name          = "cs-0"
+  prefix        = local.prefix_lnd
+  location      = var.location_config.region
+  storage_class = "REGIONAL"
+  # retention_policy = local.lnd_bucket_retention_policy
+  encryption_key = try(local.service_encryption_keys.storage != null, false) ? try(local.service_encryption_keys.storage, null) : null
```

Comment:
As a rule, can you break long lines?

I have a hard time understanding that condition, is the below one equivalent? It's definitely more legible.

```hcl
encryption_key = try(local.service_encryption_keys.storage, null)
```